services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -425,8 +401,6 @@ def update_stac_item_publication(  # pylint: disable=too-many-locals
                 old_bucket_arr[2] = CATALOG_BUCKET
                 s3_key = "/".join(old_bucket_arr)
                 # Check if the S3 key exists
-                # TBD: The catalog should have nothing to do anymore with the
-                # s3 level. This part has to be removed
```

Comment:
Should this part be remove in this story or another future story ?

Reply:
It will remain in fact